### ISSUE TITLE
[Deque Analysis] Browse by Author "serious" issues

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#1176]
> Serious: This issue results in serious barriers for people with disabilities, and will partially prevent them from accessing fundamental features or content. People relying on assistive technologies will experience significant frustration as a result. Issues falling under this category are major problems, and remediation should be a priority. 

## Deque Analysis Summary

Browse by Author pages have a total of 4 serious issues (not including color contrast which are handled in other tickets):

1. ~~(ID 469380) "Selected/Active state of the element is missing" (in the gear dropdown where "Results per page" and "Sort options" are visible).  In the dropdown, the selected RPP and Sort by options _are shown with a visible checkmark_, but that is NOT displayed to screen readers.~~
    * ~~HOW TO FIX: Use the `aria-current` attribute to indicate the element that represents the currently selected item within a container or set of related elements, such as the currently selected page, step, location, date, or time. https://www.w3.org/TR/wai-aria-1.1/#aria-current~~
    * ~~_ALSO occurs on Community Homepage, so this should be fixed in a general location._ See also ID 469634~~
    * FIXED BY #2717
2. (IDs 469483, 469487, 469491) Issues with placeholder text "Type the first few letters and click on the Browse button"
      * First, it is cut off when text spacing is adjusted
      * ~~Second, it is also cut off when you zoom 200%~~ (No longer reproducible in 7.6.1)
      * Third, it is also cut off when you set the screen at 320px width.
      * In all three scenarios that placeholder text is too long to display.  I wonder if we should simply move it to a *label* or *hint* (below the textbox) so that it can more easily wrap...instead of using placeholder text.
      
Full list of issues is viewable at (requires login): https://axeauditor.dequecloud.com/test-run/0856438a-a19a-11eb-bc31-b7d5be387c86/issues?activeTab=dt-issue&page=0&pageSize=50&sortField=ordinal&sortDir=asc&row=9&filter%5Bseverity%5D=3&filter%5Btype%5D=issue&filter%5Bpage_number%5D=2

## More Information / Tools

* Consider using [Deque's free Chrome plugin](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) to check your work

**Comment by Andrea-Guevara** [comment]
Hello, my name is Andrea Guevara and I work at Neki-it. We've analyzed the issue, reproduced the problem, following the suggestions proposed, and implemented the solution. I would like to request this issue for us.

**Comment by Andrea-Guevara** [comment]
Hello, my name is Andrea Guevara and I work at Neki-it. We've analyzed the issue, reproduced the problem, following the suggestions proposed, and implemented the solution. I would like to request this issue for us.

**Comment by Andrea-Guevara** [comment]
Good evening @tdonohue!

The correction for the issue follows this PR https://github.com/DSpace/dspace-angular/pull/3067.

Waiting feedback!
